CMakeLists.txt

```diff
@@ -14,15 +14,21 @@
 # along with roboptim-core.  If not, see <http://www.gnu.org/licenses/>.
 
 # Requires at least CMake 2.8 to configure the package.
-CMAKE_MINIMUM_REQUIRED(VERSION 2.8)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.16)
 
-INCLUDE(cmake/base.cmake)
-INCLUDE(cmake/boost.cmake)
-INCLUDE(cmake/cpack.cmake)
+SET(CMAKE_CXX_STANDARD 11)
 
 SET(PROJECT_NAME roboptim-core-plugin-ipopt)
 SET(PROJECT_DESCRIPTION "RobOptim core IPOPT plug-in")
 SET(PROJECT_URL "https://github.com/roboptim/roboptim-core-plugin-ipopt")
+set(PROJECT_USE_CMAKE_EXPORT TRUE)
+
+INCLUDE(cmake/base.cmake)
+INCLUDE(cmake/boost.cmake)
+INCLUDE(cmake/cpack.cmake)
```

Comment:
you removed `SETUP_PROJECT_CPACK()`, so I think there is no need to add this `INCLUDE(cmake/cpack.cmake)`

Reply:
Fixed by 0ba76bb